flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -223,6 +224,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>cassandra</artifactId>
```

Comment:
do we still need the cassandra-all dependency? AFAICT the connector should provide everything we need.

Reply:
Good point, thanks!
I think you're right only the driver should be needed. I'll test